### PR TITLE
@juliusv simple update to doc page for link to puppet module

### PR DESCRIPTION
**Pull request by abuxton**

### CODE REVIEW

File: content/docs/introduction/install.md [hunk 1]
```diff
@@ -103,3 +103,7 @@ Chef:
 SaltStack:
 
 * [bechtoldt/saltstack-prometheus-formula](https://github.com/bechtoldt/saltstack-prometheus-formula)
+
+Puppet:
```
Comment (brian-brazil) [inline content/docs/introduction/install.md]: P sorts before S, so this should go up one

File: content/docs/introduction/install.md [hunk 2]
```diff
@@ -103,3 +103,7 @@ Chef:
...
+
+* [https://forge.puppet.com/brutus777/prometheus](https://forge.puppet.com/brutus777/prometheus)
```
Comment (brian-brazil) [inline content/docs/introduction/install.md]: The others link to github repos, does that also make sense here?

**Comment by grobie** [comment]
@abuxton Can you fix the comments?

**Comment by brian-brazil** [comment]
This is stale, and we already have a puppet example listed.

**Pull request closed without merging**